pkg/operator/operator.go

```diff
@@ -218,5 +224,6 @@ func (optr *Operator) maoConfigFromInfrastructure() (*OperatorConfig, error) {
 			NodeLink:           machineAPIOperatorImage,
 			MachineHealthCheck: machineAPIOperatorImage,
 		},
+                BaremetalPodConfig: baremetalImages, 
```

Comment:
can we call this BaremetalControllers fo consistency?

Reply:
> can we call this BaremetalControllers fo consistency rather than BaremetalPodConfig?

Done.